files/moci-device-quarantine.sh

```
@@ -0,0 +1,194 @@
+	is_enabled_flag "$FEATURE_ENABLED" && is_enabled_flag "$QUARANTINE_ENABLED"
+}
+
+collect_leases() {
+	if [ ! -f "$LEASES_FILE" ]; then
+		return 0
+	fi
+	awk '{ if ($2 ~ /^([0-9A-Fa-f]{2}:){5}[0-9A-Fa-f]{2}$/) print tolower($2) "|" $3 "|" $4 }' "$LEASES_FILE"
+}
+
+rule_name_for() {
+	local mac="$1"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)